samples/host_exerciser/host_exerciser_cmd.h

```diff
@@ -174,6 +174,12 @@ class host_exerciser_cmd : public test_command
 
 	fpgaperf::ptr_t perf(nullptr);
         if (host_exe_->perf_) {
+            uid_t uid = getuid();
+            if (uid != 0) {
+                std::cout <<"\nFailed to read Perf counter due to unprivileged user access"<<std::endl
+                <<"check --help for more information on setting the capabilities for binary\n" <<std::endl;
+                return -1;
+            }
```

Comment:
Can't we still run but without perf counters?

Reply:
That would mean (in effect) editing the command line to the test.  Is this ok from a testing perspective, or is it better to error out and return a non-zero value?

Reply:
Perhaps in the long term we can enhance it to make performance counters optional via command line and this check